src/index/am_build.rs

```diff
@@ -57,13 +57,21 @@ unsafe extern "C" fn callback(
     _tuple_is_alive: bool,
     state: *mut std::os::raw::c_void,
 ) {
+    let state = &mut *(state as *mut Builder);
+    if *_is_null.add(0) {
```

Comment:
Undefine is for suppressing `dead_code` warning. Since we use it now, remove it.